onnxruntime/core/providers/webgpu/tensor/concat.h

```diff
@@ -24,6 +24,20 @@ class ConcatProgram final : public Program<ConcatProgram> {
   size_t axis_;
 };
 
+class ConcatProgramSingle final : public Program<ConcatProgramSingle> {
+ public:
+  ConcatProgramSingle(size_t axis) : Program{"Concat"}, axis_{axis} {
```

Comment:
use "ConcatSingle" to differentiate between this and the old concat program class.